src/main/java/de/dennisguse/opentracks/settings/PreferencesUtils.java

```diff
@@ -147,6 +147,14 @@ private static boolean getBoolean(int keyId, boolean defaultValue) {
     }
 
 
+    public static String getHeight() {
+        return getString(R.string.settings_profile_height_key,"" );
+    }
+
+    public static void setHeight(String newHeight) {
+        setString(R.string.settings_profile_height_key, newHeight);
+    }
+
```

Comment:
The requirement:
Height: (feet or metric depending Settings->Default Units…)

=> You might need to also store the unit alongside the number, then in the getter function, you'll need to check the current default unit. If they both match, then return the number. If not, you'll need to perform unit conversion to convert the number to the correct unit.

Use getUnitSystem() in PreferencesUtils to get the current setting.